studio/components/opendax/AccountButton/AccountModal.tsx

```diff
@@ -5,126 +5,115 @@ import React, { FC, Fragment, useRef } from 'react'
 import { ExplorerLink } from '../ExplorerLink'
 
 type AccountModalProps = {
-    open: boolean
-    setOpen: (open: boolean) => void
-    address: string
-    chain: Chain
-    handleLogout: () => void
+  open: boolean
+  setOpen: (open: boolean) => void
+  address: string
+  chain: Chain
+  handleLogout: () => void
 }
 
 export const AccountModal: FC<AccountModalProps> = ({
-    open,
-    setOpen,
-    address,
-    chain,
-    handleLogout,
+  open,
+  setOpen,
+  address,
+  chain,
+  handleLogout,
 }: AccountModalProps) => {
-    const ctaButtonRef = useRef(null);
+  const ctaButtonRef = useRef(null)
 
-    return (
-        <Transition.Root show={open} as={Fragment}>
-            <Dialog
-                as="div"
-                static
-                className="fixed z-40 inset-0 overflow-y-auto"
-                initialFocus={ctaButtonRef}
-                open={open}
-                onClose={setOpen}
-            >
-                <div className="flex md:items-center items-end justify-center min-h-screen pt-4 px-0 md:px-4 pb-0 md:pb-20 text-center">
-                    <Transition.Child
-                        as={Fragment}
-                        enter="ease-out duration-300"
-                        enterFrom="opacity-0"
-                        enterTo="opacity-100"
-                        leave="ease-in duration-200"
-                        leaveFrom="opacity-100"
-                        leaveTo="opacity-0"
-                    >
-                        <Dialog.Overlay className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
-                    </Transition.Child>
+  return (
+    <Transition.Root show={open} as={Fragment}>
+      <Dialog
+        as="div"
+        static
+        className="fixed z-40 inset-0 overflow-y-auto"
+        initialFocus={ctaButtonRef}
+        open={open}
+        onClose={setOpen}
+      >
+        <div className="flex md:items-center items-end justify-center min-h-screen pt-4 px-0 md:px-4 pb-0 md:pb-20 text-center">
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <Dialog.Overlay className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
+          </Transition.Child>
 
```

Comment:
i feel that we do not need this component anymore;
we just need to display a white screen while checking the session